### DISCUSSION TITLE
Difference of decimation algorithm between this and MeshDecimator

### DISCUSSION THREAD

**Discussion by thegreatgramcracker** [Whinarn/UnityMeshSimplifier#66] [Q&A] [answered]
Hello, I know this project isn't being actively worked on, but I am currently using the Mesh Decimator library for a small modding project to convert models to low poly versions. I'm aware that this is primarily for LODs, but I was able to get pretty satisfying results using the lossless decimation option, but there is some reoccurring texture distortion present. In the MeshDecimator github page, it says that this project has an improved version of the algorithm. I'm wondering if this is mainly just performance improvements or has the algorithm been made to be more intelligent? I don't need a guarantee, I just wanted to know if the possibility is worth checking. I can rework my program to convert models to Unity format and then use this library to decimate and convert back to the format I need, but I wanted to ask if there's even any chance that it will yield better results.

**Comment by Whinarn** [comment, marked as answer]
It was so long ago that I deprecated MeshDecimator, I cannot recall accurately. But for sure most improvements are in terms of performance. I do believe that there are improvements related to UVs also however. There has been plenty of changes since then. I would check the changelog and match the date of the last release of MeshDecimator to figure out what changed in detail. I hope that helps!

**Reply by thegreatgramcracker** [reply]
That's enough cause for me to look in to it further, thanks for the response!
